src/Kernels/Plasticity.h

```diff
@@ -42,42 +44,42 @@
 #ifndef KERNELS_PLASTICITY_H_
 #define KERNELS_PLASTICITY_H_
 
-#include "Model/plasticity.hpp"
+#include "Initializer/BatchRecorders/DataTypes/ConditionalTable.hpp"
 #include "Initializer/typedefs.hpp"
-#include "generated_code/tensor.h"
+#include "Model/plasticity.hpp"
```

Comment:
When do we want a file to start with a capital letter, and when not?

Reply:
Actually, there is no convention right now. :) Just the majority tends to use uppercase letters, thanks to the former FORTRAN influence of SeiSsol. But sometimes, folks choose to use lowercase (e.g. out of habit, and the reviewers not complaining because it's hard to notice when you're not looking for it).

That said, we don't have yet established a (written) rule on what convention (uppercase/lowercase) to follow or to ultimately adjust to. Same goes for the file ending—some prefer to use `.h`, some `.hpp`.

I've left it open for a future PR to handle (once a convention is established for naming style and file ending).